app-frontend/src/app/components/scenes/sceneItem/sceneItem.html

```diff
@@ -2,11 +2,18 @@
   <div ng-if="$ctrl.isDraggable" ui-tree-handle>
     <i class="icon-gripper"></i>
   </div>
-  <rf-status-tag entity-type="scene" status="$ctrl.scene.statusFields.ingestStatus"></rf-status-tag>
-  <img ng-if="$ctrl.scene.thumbnails[0]"
+  <rf-status-tag entity-type="scene"
```

Comment:
This seems to have a lot of duplicate logic
I'd rather have a containing 
`<div ng-if="$ctrl.isRfScene">` and `<div ng-if=!$ctrl.isRfScene">`
 then the images etc inside it